modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/frontend/data/set/filter/factory/ObjectFieldFDSFilterFactoryRegistryUtil.java

```diff
@@ -61,8 +58,24 @@ public ObjectFieldFDSFilterFactory getObjectFieldFDSFilterFactory(
 			objectField.getBusinessType());
 	}
 
-	@Activate
-	protected void activate(BundleContext bundleContext) {
+	public ObjectFieldFDSFilterFactoryRegistryUtil(
+		ObjectFieldLocalService objectFieldLocalService) {
+
+		_objectFieldLocalService = objectFieldLocalService;
```

Comment:
Maybe passing the class directly in the method (the only usage) is an alternative 🤔